### PR TITLE
Setup

### PR DESCRIPTION
**Pull request by christianonoh**
Pull Request: Add Database Schema, Data, and Queries

Description:
This pull request adds the database schema, inserts data into the "animals" table, and includes queries to retrieve specific information from the database. It also includes a screenshot of the results of the queries.

Screenshots:
Screenshot 2023-06-06 at 6 56 29 PM

Screenshot 2023-06-06 at 9 02 54 PM

Please review and give any feedback.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,34 +152,16 @@ If you like this project...
...
-
-  - [Answer_2]
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (t-yanick) [inline README.md:162]: - [ ] Great job working on this milestone and updating the ReadMe. Unfortunately, when I click on the MIT license link, it gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue.
image

**Review by t-yanick — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED

Hi @christianonoh , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

✔️ SQL commands put in the correct files
✔️ Screenshots added to PR description
✔️ Descriptive Readme file

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Great job adding screenshots to your PR description. At the moment, you are missing screenshots of 3 queries. Kindly run these queries and add the screenshots
- Find all animals that its name ends in "mon".
- List the name of all animals born between 2016 and 2019.
- List the name of all animals that are neutered and have less than 3 escape attempts.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (alexander16108) [inline README.md:162]: as suggested previously your MIT license link gives me an error 404 page. It is not professional to allow broken links in our codebase. Kindly fix this issue 😄 
image

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @christianonoh , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
✔️ SQL commands put in the correct files
✔️ Screenshots added to PR description
✔️ Descriptive Readme file

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -100,137 +58,73 @@ After you're finished please remove all the comments and instructions!
...
-- **[key_feature_3]**
+- **Has an animals table schema**
+- **Has animals table data**
+- **A couple of animals search queries**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (tufoinnkuo10) [inline README.md:69]: - [optional] The live demo section can be removed since the projects is not deployed. This will keep your README file clean.

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @christianonoh  

Your project is complete! Great job🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request✔️
- commands well placed✔️
- screenshots ok ✔️ 



----[optional] please always make your pull request title descriptive enough so from a glance they will know what you worked on.
<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**